packages/strapi-plugin-content-type-builder/admin/src/components/LiTableList/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+*
+* LiTableList
```

Comment:
Why did you choose this name? I'm not a big fan because the name is too explicit and use a DOM element into it. It would be better to describe what it looks like or describe his behavior.